models/classes/controllerMap/ActionDescription.php

```diff
@@ -34,8 +32,15 @@
  */
 class ActionDescription
 {
-    /** @var DocBlock|null */
-    private static $docBlock;
+    /**
+     * Data cache for DocBlock information used by getDocBlock().
+     *
+     * The method name the docblock is found in is used as the array index
+     * (i.e. it works as a "cache key")
+     *
+     * @var DocBlock[]
+     */
+    private static $docBlock = [];
```

Comment:
Maybe, since it has a collection now it should be plural? Do we really need to explain this variable in the annotation? Isn't self-explainable in the code?

```suggestion
    /** @var DocBlock[] */
    private static $docBlocks = [];
```